tests/test_binary.nim

```diff
@@ -7,10 +7,12 @@ const
   binaryName = "configlet" & binaryExt
 
 proc main =
-  let repoRootDir = getAppDir().parentDir()
+  let repoRootDir = getCurrentDir()
```

Comment:
```suggestion
  const repoRootDir = currentSourcePath.parentDir().parentDir()
```

See https://nim-lang.github.io/Nim/system.html#currentSourcePath.t

Does this work for you? During development, I like to be able to run the tests from any directory.

Reply:
Yes, that works nicely. It fixes the path issue